### PR TITLE
measure size of data directory after test

### PR DESCRIPTION
**Pull request by jeffreyscarpenter**
Add a simple way of measuring the size of data written during Nosqlbench performance tests - shell into the Stargate coordinator (dev mode), find the data directory, and calculate its size.

**Checklist**
- [ ] Changes manually tested
- [x] Automated Tests added/updated
- [ ] Documentation added/updated
- [x] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: .github/workflows/performance-testing.yaml [hunk 1]
```diff
@@ -16,6 +16,9 @@ on:
         required: true
         type: string
         default: '20'
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '0 4 * * *'
```
Comment (jeffreyscarpenter) [inline .github/workflows/performance-testing.yaml:21]: note this schedules the tests to run nightly at 4am GMT

File: .github/workflows/performance-testing.yaml [hunk 2]
```diff
@@ -108,6 +117,13 @@ jobs:
           export AUTH_TOKEN=$(curl -s -X POST 'http://localhost:8081/v1/auth' -H 'Content-Type: application/json' --data-raw '{"username": "cassandra", "password": "cassandra"}' | jq -r '.authToken')
           ../nb5 -v --report-csv-to logs --log-histograms logs/hdr_metrics.log ${{ matrix.test }} jsonapi_host=localhost auth_token=$AUTH_TOKEN docscount=${{ inputs.docscount }} connections=${{ inputs.connections }}
 
+      # Simple approach measuring size of data directory
+      - name: Collect Data Size Stats
+        run: |
+          cd docker-compose
+          DATA_DIR=$(docker-compose exec -T coordinator find /tmp -name "data")
+          echo 'Data size:' $(docker-compose exec -T coordinator du -sh "$DATA_DIR/${{ matrix.keyspace }}") >> $GITHUB_STEP_SUMMARY
```
Comment (jeffreyscarpenter) [inline .github/workflows/performance-testing.yaml:125]: the data size shows up on the summary page for the workflow run. we can likely make use of this for additional stats and get a bit more advanced with the output (GH actions accepts markdown format).

Comment (ivansenic) [reply .github/workflows/performance-testing.yaml:125]: so test deletes all the data as the last step.. what would this show us?

File: .github/workflows/performance-testing.yaml (hunk 1 above)
Comment (tatu-at-datastax) [reply .github/workflows/performance-testing.yaml:21]: Would make sense to include that in comment above?

**Review by tatu-at-datastax — APPROVED** [review]

File: .github/workflows/performance-testing.yaml (hunk 2 above)
Comment (ivansenic) [reply .github/workflows/performance-testing.yaml:125]: Would be also cool to know how many documents were inserted in the test, I just see: `Data size: 12K /tmp/stargate-DSE Stargate Backend2162680328132524095/data/jsonapi_crud_basic`

Comment (jeffreyscarpenter) [reply .github/workflows/performance-testing.yaml:127]: added docscount to output

**Pull request merged**